source/manual/on-call.html.md

```diff
@@ -23,6 +23,22 @@ to keep GOV.UK running at night, on the weekends and on public holidays.
 - Logs are not as important as being available - if you need to lose some logs
   in order to bring the site back up, that's probably a good trade-off to make
 
+## On call checklist
+
+You should do these things before going on call so you're prepared.
+
+1. Have the numbers of other people on your shift saved in your phone. This
```

Comment:
Possible talk about looking at your alert settings in pager duty and indeed maybe installing the pager duty app.

Reply:
Good shout, done.